packages/react-native-jigsaw/src/components/Layout.tsx

```diff
@@ -0,0 +1,324 @@
+import * as React from "react";
+import { View, StyleProp, ViewStyle } from "react-native";
+// @ts-ignore
+import type { ViewStyleProp } from "react-native/Libraries/StyleSheet/StyleSheet";
+
+import {
+  GROUPS,
+  FORM_TYPES,
+  PROP_TYPES,
+  COMPONENT_TYPES,
+} from "../core/component-types";
+
+export function Center({
+  width,
+  height,
```

Comment:
Why pass `width` and `height` as props but also a style object? Should they just live in `style`?

Reply:
trying to make the api as small as possible, wdyt?

Reply:
works 4 me